tests/TestCase/ORM/Behavior/TranslateBehaviorTest.php

```diff
@@ -1847,7 +1854,7 @@ public function testBuildMarshalMapBuildEntitiesValidationErrors()
                 '_empty' => 'This field cannot be left empty',
             ],
         ];
-        $this->assertEquals($expected, $entity->getError('es'));
+        $this->assertEquals($expected, $entity->getError('_translations.es'));
```

Comment:
This seems like the kind of change that could cause problems for userland code as the key errors are stored in is changing. This does match up with the documentation better now, but I suspect we might still get an issue about the change.

Reply:
With the current array structure the errors wouldn't be displayed under the related fields using `FormHelper::control()`.

There's a possibility the someone could be using `FormHelper::error()` to fetch and display the errors at appropriate location, but that seems quite unlikely. 

That said I can move the error key change to 4.next if we want to be extra safe.

Reply:
I'm fine either way. We should think about what our response will be if we get issues that the behavior 'broke'.

Reply:
Meh, I will just move the array keying fix to 4.next :)

Reply:
Removed the error key changes and just kept the cleanup.